gatsby-config.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
     siteMetadata: {
         title: `OpenUI5`,
         description: ``,
+        faviconUrl: `/favicon/favicon.png`,
```

Comment:
where is the configuration that favicons are going to genegerated in `favicon/` folder?

Reply:
This is the static file without background. gatsby-plugin-manifest plugin supports only 1 image for generation. For this reason I used image with white background to generate the all sizes. Favicon is not generated with gatsby-plugin-manifest plugin: line 30

Reply:
As discussed, lets keep all generated icons in `favicon` folder for consistency

Reply:
done